src/services/events/hook/useEvents.ts

```
@@ -4,10 +4,10 @@
 import * as api from '@/services/events/api'
 import { PAGINATED_QUERY_FEED } from '@/services/timeline-feed/hook/index'
 import { useToastController } from '@tamagui/toast'
-import { useMutation, useQueryClient, useSuspenseInfiniteQuery, useSuspenseQuery } from '@tanstack/react-query'
+import { Query, useMutation, useQueryClient, useSuspenseInfiniteQuery, useSuspenseQuery } from '@tanstack/react-query'
```

ESLint

[error] 'Query' is defined but never used.

(unused-imports/no-unused-imports)